classes/class-wc-connect-nux.php

```diff
@@ -546,6 +545,44 @@ public function show_banner_after_connection() {
 			) );
 		}
 
+		public function show_tos_banner() {
+			if ( ! $this->should_display_nux_notice_for_current_store_locale() ) {
+				return;
+			}
+
+			if ( ! $this->should_display_nux_notice_on_screen( get_current_screen() ) ) {
+				return;
+			}
+
+			if ( isset( $_GET['wcs-nux-tos'] ) && 'accept' === $_GET['wcs-nux-tos'] ) {
+				WC_Connect_Options::update_option( 'tos_accepted', true );
+
+				$this->tracks->opted_in( 'tos_banner' );
+
+				wp_safe_redirect( remove_query_arg( 'wcs-nux-tos' ) );
+				exit;
+			}
+
+			$country = WC()->countries->get_base_country();
+			/* translators: %s: list of features, potentially comma separated */
+			$description_base = __( "WooCommerce Services is almost ready to go! Once you connect your store you'll have access to %s.", 'woocommerce-services' );
+			$feature_list     = $this->get_feature_list_for_country( $country );
+
+			$this->show_nux_banner( array(
+				'title'          => __( 'Connect your store to activate WooCommerce Services', 'woocommerce-services' ),
+				'description'    => esc_html( sprintf( $description_base, $feature_list ) ),
+				'button_text'    => __( 'Connect', 'woocommerce-services' ),
+				'button_link'    => add_query_arg( array(
+					'wcs-nux-tos' => 'accept',
+				) ),
+				'image_url'      => plugins_url(
+					'images/wcs-notice.png', dirname( __FILE__ )
+				),
+				'should_show_jp'    => true,
```

Comment:
Question: do we want to display the JP logo in this banner? We didn't used to display it here, and I'm not really sure it's needed.

Reply:
Whoops! Good catch - that was a copy-paste error I made.

Reply:
Fixed in https://github.com/Automattic/woocommerce-services/pull/1190/commits/60954b9281d0d80700250276a076bd42663cf7ef.